.idea/codeStyles/Project.xml

```diff
@@ -1,8 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <AndroidXmlCodeStyleSettings>
-      <option name="ARRANGEMENT_SETTINGS_MIGRATED_TO_191" value="true" />
-    </AndroidXmlCodeStyleSettings>
```

Comment:
Please remove the project file changes from this PR. We need to figure out how to avoid the unwanted changes to checked in files in a separate PR.

Reply:
Could you help me how can I do this ?

Reply:
See suggestions here: https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

Reply:
Just want to confirm with you before making any changes -
commit hash when first time this file was added: 254916d6e23c1247bae77620995d7261a3fba1ea

1. `git reset HEAD^ 254916d6e23c1247bae77620995d7261a3fba1ea -- .idea/codeStyles/Project.xml`
1. `git commit --amend --no-edit`

or

1.  `git checkout 254916d6e23c1247bae77620995d7261a3fba1ea -- .idea/codeStyles/Project.xml`
1. `git commit ...`

which one should I do ?

Reply:
Can you update Android Studio to the latest version? This should add the code back.

Reply:
So I've to keep the changes made by IDE itself? that would be sufficient?

Reply:
My Android Studio is up-to-date.

Android Studio 3.5
Build #AI-191.8026.42.35.5791312, built on August 9, 2019
JRE: 1.8.0_202-release-1483-b03 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Windows 10 10.0

Reply:
> So I've to keep the changes made by IDE itself? that would be sufficient?

My Android Studio added this after updating to the latest version. It must not be changed in this PR.

Reply:
But What am I supposed to do here? If you need these lines of code, I could simply copy paste and commit the same.

Reply:
Just do not change them in this PR. Whether you want to remove them from your existing commits and force push the PR branch, or whether you add a commit that restores them, is up to you, it's fine either way.

Reply:
Actually it was changed in one of the first commits of this PR and I've squashed the changes in single commit before opening this PR. So its a bit difficult to identify that particular commit so that I could revert this file to that version.